code/src/containers/forms/product-form/components/main-info/MainInfo.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+
+import {
+  ProductFormFieldErrors,
+  ProductFormRegisterFunction
+} from "@/containers/forms/product-form/ProductForm.types";
+import MainInfo from "@/containers/forms/product-form/components/main-info/MainInfo";
+
+import getTagIn from "@/utils/get-tag-in/getTagIn";
+
+const errors = {
+  productTranslations: [
+    {
+      name: {
+        message: "Name error message"
+      },
+      description: {
+        message: "Description error message"
+      }
+    },
+    {
+      name: {
+        message: "Name error message"
+      },
+      description: {
+        message: "Description error message"
+      }
+    }
+  ]
+} as ProductFormFieldErrors;
+
+const rooteError = {
+  productTranslations: {
+    root: {
+      message: "Root error message"
+    }
+  }
+} as ProductFormFieldErrors;
+
+const registerFunction = (() => ({})) as unknown as ProductFormRegisterFunction;
+
+describe("Test MainInfo component", () => {
+  test("Should display language buttons", () => {
+    render(<MainInfo register={registerFunction} errors={errors} />);
+
+    const languageButtons = screen.getAllByTestId(
+      "product-form-language-button"
+    );
+
+    expect(languageButtons).toHaveLength(2);
+  });
+
+  test("Should navigate between languages", () => {
+    render(<MainInfo register={registerFunction} errors={errors} />);
```

Comment:
It is repeated in a bunch of tests. Please fix it

Reply:
Thank you, fixed